skimage/morphology/extrema.py

```diff
@@ -0,0 +1,349 @@
+"""extrema.py - local minima and maxima
+
+This module provides functions to find local maxima and minima of an image.
+Here, local maxima (minima) are defined as connected sets of pixels with equal
+grey level which is strictly greater (smaller) than the grey level of all
+pixels in direct neighborhood of the connected set. In addition, the module
+provides the closely related functions h-maxima and h-minima.
+
+Soille, P. (2003). Morphological Image Analysis: Principles and Applications
+(2nd ed.). Springer-Verlag New York, Inc.
+
+Original author: Thomas Walter
```

Comment:
Sorry. I did not want to get particular credits, but I had seen that in other files. I remove this of course.